cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -360,8 +360,28 @@ func (c *Client) RemoveMember(id uint64) ([]Member, error) {
 			defer func() { _ = cli.Close() }()
 
 			ctx, cancel := context.WithTimeout(context.Background(), etcdTimeout)
+			defer cancel()
+
+			// list members and quick return if the member don't exist
+			listResp, err := cli.MemberList(ctx)
+			if err != nil {
+				klog.V(5).Infof("Failed to check whether the member %s exists, %v", strconv.FormatUint(id, 16), err)
```

Comment:
```suggestion
				klog.V(5).Infof("Failed to check whether the member %q exists: %v", strconv.FormatUint(id, 16), err)
```

Reply:
trying to use %q everywhere since the old code has %q
didn't check if there are %s cases..